README.md

```diff
@@ -44,7 +44,7 @@ This open source project is a work in progress and ever evolving.
 
 We welcome all contributions, suggestions and ideas for improvement from the community.
 
-You can contribute by adding  new quiz questions or making changes to the code.
+You can contribute by adding new quiz questions or making changes to the code [here](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/src/data/full-quiz.js).
```

Comment:
Proposed change:

You can contribute by fixing bugs in the codebase, proposing new features or adding new questions to the [quiz file](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/src/data/full-quiz.js).

Reply:
```suggestion
You can contribute by fixing bugs in the codebase, proposing new features or adding new questions to the [quiz file](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/src/data/full-quiz.js).
```